less/common/sideNav.less

```diff
@@ -30,8 +30,8 @@
           & .Button-icon {
             float: left;
             margin-left: -30px;
-            margin-top: 1px;
             font-size: 15px;
+            line-height: initial;
```

Comment:
Both of the `line-height: initial` rules are not actually doing anything for me. Can we remove them?

Reply:
Hm, sure. I remember seeing a slight change with the `line-height` rules, but I can no longer reproduce them.